internal/api/schemas/v1.yaml

```diff
@@ -11,17 +11,168 @@ info:
     name: Mozilla Public License 2.0
     url: https://www.mozilla.org/en-US/MPL/2.0/
 paths:
+  "/namespaces/{namespace}/gateways":
+    get:
+      description: 'Returns all configured gateways in the given namespace.'
+      operationId: listNamespacedGateways
+      parameters:
+      - name: namespace
+        in: path
+        description: Namespace of gateways to fetch
+        required: true
+        schema:
+          type: string
+      responses:
+        '200':
+          description: gateway response
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/GatewayPage"
+        default:
+          description: unexpected error
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/Error"
   "/gateways":
```

Comment:
I'm not sure I'd add this alias for the default namespace just yet - it feels like we're increasing our API surface area right off the bat when it may be unnecessary.

Honestly, I don't have any strong cons in my head other than the increased surface area, I just default to being very conservative with API design since we own it ~~forever~~ for a long time once it's out in the wild. Thoughts?

Reply:
Based on the query parameter below though, I see you're thinking of this as a way to query gateways across 2+ namespaces as well, so that may be a required feature that justifies the surface area.

Reply:
Yeah, so this would basically allow you to request either from default, or unscope from all namespaces -- since we _are_ going to have to support putting gateways/routes in multiple namespaces from the get-go, I'm fine with aliasing up-front.

Reply:
I believe we will need to list gateways in all namespaces. Are you thinking you reserve a keyword or something for that like `GET /gateways?namespaces=ALL`?

Reply:
That's the thought, maybe making something like `*` the keyword just since no one is going to use that as the name of a namespace.